PRIVACY.md

```diff
@@ -236,6 +238,12 @@ The Semgrep team uses this to answer the following questions:
   We use this to evaluate experimental features
   and understand their production-readiness.
 
+> Engine requested (OSS, Pro, Interfile)
```

Comment:
same sort of deal here. i think i just regard "what engines people are using" as rather casual language